src/aoc2020/day2_copy.clj

```diff
@@ -0,0 +1,146 @@
+(ns aoc2020.day2_copy
+  (:require [aoc2020.util :refer [file->seq]])
+  (:require [clojure.string :as str]))
+
+(defn parsing-policy-pwd
```

Comment:
odd tense? Wouldn't this be either 
* noun form - `parsed-pwd-policy`
* verb form - `parse-pwd-policy`